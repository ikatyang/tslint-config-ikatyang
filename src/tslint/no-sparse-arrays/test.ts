// good

[1, undefined, 3];

// bad

[1, , 3];
