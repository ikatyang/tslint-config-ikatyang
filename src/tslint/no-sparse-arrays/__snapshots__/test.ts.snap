// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
// good

[1, undefined, 3];

// bad

[1, , 3];
   ~      [Array has a missing element.]

`;
